doc/content/guides/connecting-gateways/ciscoixm/_index.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Cisco LoRaWAN Gateway"
```

Comment:
Is it not called IXM or something? Please use their product name for SEO

Reply:
it is gateway is called `Cisco Wireless Gateway for LoRaWAN` but the models are called 
`IXM-LPWA-800-16-K9` & `IXM-LPWA-900-16-K9`